crates/compiler/src/builtins.rs

```diff
@@ -951,18 +949,18 @@ fn mk_builtin_table() -> Vec<Builtin> {
     ]
 }
 
-pub fn make_builtin_labels() -> HashMap<Symbol, Name> {
+pub fn make_builtin_offsets() -> HashMap<Symbol, usize> {
     let mut b = HashMap::new();
-    for (i, builtin) in BUILTIN_DESCRIPTORS.iter().enumerate() {
-        b.insert(builtin.name, Name(i as u16));
+    for (offset, builtin) in BUILTIN_DESCRIPTORS.iter().enumerate() {
+        b.insert(builtin.name, offset);
     }
 
     b
 }
-pub fn make_labels_builtins() -> HashMap<Name, Symbol> {
+pub fn make_offsets_builtins() -> HashMap<usize, Symbol> {
```

Comment:
Nit: introduce a newtype or maybe a type alias `BuiltinOffset` instead of using a naked `usize` (to the tune of `UnboundName`)?